docker/agent/Dockerfile

```diff
@@ -44,18 +44,19 @@ RUN apt-get update \
 RUN curl -LO https://packages.microsoft.com/keys/microsoft.asc && \
     echo 2cfd20a306b2fa5e25522d78f2ef50a1f429d35fd30bd983e2ebffc2b80944fa microsoft.asc| sha256sum --check - && \
     apt-key add microsoft.asc && \
-    apt-add-repository https://packages.microsoft.com/debian/11/prod && \
+    rm microsoft.asc && \
```

Comment:
Does it matter that this is no longer `-f`?

Reply:
Maybe the other rm requires it and not this one. I tested it on 3 machines (x64 and arm64) and it worked fine.